tpl/hugo/init.go

```diff
@@ -11,7 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package site
+// Package hugo provides template functions for accessing the Site Hugo object.
```

Comment:
What is a "Site Hugo" object?

I'm not sure what the correct term is, but that isn't. And this doesn't really "provide template functions accessing ..."

Reply:
How about `Package hugo provides easy access to Hugo version information.`?

I'm working on generating template namespace docs for the docs repo, and I'm using the package godoc as the namespace description.

Reply:
BTW, the docs generation is a one-time event right now.  I don't intend to update the docs site continually with what I'm doing.  Just writing a bash script to do the heaving lifting.